alf/experience_replayers/replay_buffer.py

```diff
@@ -90,6 +99,24 @@ def __init__(self,
             allow_multiprocess (bool): whether multiprocessing is supported.
             keep_episodic_info (bool): index episode start and ending positions.
                 If None, its value will be set to True if ``num_earliest_frames_ignored``>0
+            record_episodic_return (bool): If True, computes and stores episodic return for
+                every step in the episode upon episode completion.  The field
+                ``discounted_return'' stores the information.  When episodes are
```

Comment:
'' -> ``. This is a minor issue that appeared multiple times in this section.

Reply:
Ah, good catch.  Got mixed up with LaTex.